code/modules/gunse/modular_gun_parts.dm

```diff
@@ -681,3 +788,30 @@ ABSTRACT_TYPE(/obj/item/gun_parts/accessory)
 	icon_state = "juicer_drum"
 	overlay_y = 8
 
+/obj/item/gun_parts/magazine/soviet/tenround //no name additions for the next two. Don't change it!- eagle
+	name = "the bottom third of a AK magazine"
+	desc = "Some fuckin asshole took the other two thirds!"
+	max_ammo_capacity = 10
+	jam_frequency_reload = 10
+	icon_state = "thirdcapmag"
+	part_DRM = GUN_SOVIET
+	icon = 'icons/obj/items/modular_guns/mistfits.dmi'
+
+
+/obj/item/gun_parts/magazine/soviet/halfak
+	name = "Half a AK mag"
+	desc = "Another sawed in half AK magazine. Bummer."
+	max_ammo_capacity = 15
+	jam_frequency_reload = 5
+	icon_state = "halfcapmag"
+	part_DRM = GUN_SOVIET
+	icon = 'icons/obj/items/modular_guns/mistfits.dmi'
+
+/obj/item/gun_parts/magazine/soviet/ak
+	name = "Intact AK magazine"
+	desc = "One of the few still intact AK mags this side of the frontier! Score!"
+	max_ammo_capacity = 30 // ITS A FULL AK MAG OKAY
```

Comment:
this number is kind of absurd. 10 should be an absolute hard limit imo, and even then would be an insanely powerful roundstart rush item

Reply:
Changed the mags. They now go 3,5,8, and only can be used with soviet weapons. A full 30 round mag is avaliable for admin gimmicks and uses the same sprite as the 8 round one. Justification is they are gunked up, and can't fully load as a result.